app/assets.py

```diff
@@ -30,3 +30,23 @@
     filters='jsmin',
     output='scripts/vendor.js'
 )
+
+asylum_scss = Bundle(
+    'asylum/main.css',
+    output='styles/asylum.css'
+)
+
+asylum_css = Bundle(
+    'asylum/bootstrap.min.css',
+    'asylum/font-awesome.min.css',
+    'asylum/icons.css',
+    output='styles/asylum2.css'
+)
+
+asylum_js = Bundle(
+    'asylum/bootstrap.min.js',
+    'asylum/jquery-1.12.0.min.js',
```

Comment:
m4a uses jquery 2. asylum uses jquery 1.12.0. I'm not sure if we would see bugs if I dropped this one or not.